src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -319,7 +319,7 @@ const Notebooks = _.flow(
       div({
         style: {
           margin: '0 2.5rem 2.5rem 0', display: 'flex',
-          height: 250, width: 200, flexDirection: 'column',
+          height: 350, width: 200, flexDirection: 'column',
```

Comment:
This seems like an unrelated change and doesn't look like what we want?

Reply:
This is the height of the outer div - when it's only 250 the new button can't be very big. I don't know that this is 100% the way we want to go for the final choice (i.e. a hardcoded outer div height) but for now it allows the buttons to be big enough to see text.

Reply:
^ or that's how it seems to me and testing seems to fit this theory, but I could be misinterpreting what's controlling the button height style?

Reply:
That will change existing buttons though, it seems to stretch them a lot taller.
<img width="219" alt="Screen Shot 2020-09-03 at 1 13 41 PM" src="https://user-images.githubusercontent.com/15351021/92146162-4e977b00-ede7-11ea-8930-151ef65fa042.png">

Reply:
ah hmm I was trying to keep the pattern as similar to the existing as possible however now I'm wondering if this even should be using flex and maybe each button should just be setting their own height

Reply:
or i could just add a max-height to the buttons for now?